x/metadata/spec/examples/kotlin/src/main/kotlin/kbech32/Bech32.kt

```diff
@@ -0,0 +1,158 @@
+package io.provenance.kbech32
```

Comment:
I recommend you remove this version and use the one Figure/Provenance has already published for consistency.

Reply:
Since this is related to the comment above (https://github.com/provenance-io/provenance/pull/307#discussion_r631208944), I'll resolve this comment and we can just use that one.